### ISSUE TITLE
Feature request: search as you type / suggestions for internal links

### ISSUE THREAD

**Issue by fquadt** [trendschau/typemill#499]
It would be nice if there existed a more supporting element for creating internal links (to pages or anchors). At the moment the link to an internal page or anchor has to be obtained via the frontend and is entered like an external link. 

In the dokuwiki system as a good example you can enter a word from the page title and you are presented with a list of pages containing this keyword:
![image](https://github.com/user-attachments/assets/c96325be-3312-4666-b88b-b7c5e6fe57ba)

Or like in automad:
![image](https://github.com/user-attachments/assets/dafff6a8-1a12-4b56-bc6d-1aa8a75bc5b8)

Florian

**Comment by trendschau** [comment]
yes, thank you for the input, in fact I planned something like this already, but there is a long priority-list so I think it is something for next year

**Comment by trendschau** [comment]
Ok, I have realized now for the new link feature for prompts. I will keep it open so I can add it to other places in future, especially the link component.
